### PR TITLE
feat(sms): Return the `formattedPhoneNumber` in the POST /sms response.

### LINKED ISSUE #2176: Return the formatted phone number in the response to /sms
google-libphonenumber can format a phone number in a country's local format. The content-server can lean on the auth-server to return the formatted telephone number an SMS is sent to instead of having to code that into the content server - less code, faster development!

Supports https://github.com/mozilla/fxa-content-server/issues/5572
Ref https://github.com/mozilla/fxa-auth-server/issues/2177

### PR DESCRIPTION
**Pull request by shane-tomlinson**
formattedPhoneNumber is used by the content server to display the
telephone number the SMS was sent to, formatted for the user's country.

fixes #2176

@philbooth - r?

### CODE REVIEW

File: docs/api.md [hunk 1]
```diff
@@ -255,6 +255,8 @@ for `code` and `errno` are:
   This email can not currently be used to login
 * `code: 400, errno: 150`:
   Can not resend email code to an email that does not belong to this account
+* `code: 400, errno: 151`:
```
Comment (shane-tomlinson) [inline docs/api.md]: I'm not sure why these documentation changes ended up here, I imaging the pre-push hook kicked in.

File: test/local/routes/sms.js [hunk 2]
```diff
@@ -121,12 +122,17 @@ describe('/sms with the signinCodes feature included in the payload', () => {
         assert.equal(args[0].event, 'sms.installFirefox.sent')
         assert.equal(args[0].flow_id, request.payload.metricsContext.flowId)
       })
+
+      it('returns a formattedPhoneNumber', () => {
+        assert.equal(response.formattedPhoneNumber, '888-508-3401')
```
Comment (shane-tomlinson) [inline test/local/routes/sms.js]: The country code is not returned in the response, but we add that on the front end.

File: lib/routes/sms.js [hunk 3]
```diff
@@ -50,7 +50,7 @@ module.exports = (log, db, config, customs, sms) => {
         const templateName = TEMPLATE_NAMES.get(request.payload.messageId)
         const acceptLanguage = request.app.acceptLanguage
 
-        let phoneNumberUtil, parsedPhoneNumber
+        let phoneNumberUtil, parsedPhoneNumber, region
```
Comment (philbooth) [inline lib/routes/sms.js]: No biggie but is there a reason I don't see for moving `region` up to this level? It doesn't look like it's used?

File: docs/api.md (hunk 1 above)
Comment (philbooth) [reply docs/api.md]: Yep, feel free to submit them as a separate PR if you don't want them stinking up this changeset, or not if you don't mind them.

File: lib/routes/sms.js (hunk 3 above)
Comment (shane-tomlinson) [reply lib/routes/sms.js]: Ahha! You are right. I used a different formatter (FormatAsYouType or something like that) to begin with, and that required `region`. Will revert that change.

**Pull request merged**